srcpkgs/mathematica/template

```diff
@@ -0,0 +1,52 @@
+# Template file for 'mathematica'
+pkgname=mathematica
+version=13.0.0
+revision=1
+archs="x86_64"
+# depends="libfont fontconfig alsa-lib libXext libglvnd xdg-user-dirs"
+makedepends="tar xz"
+short_desc="Computational software"
+maintainer="dkwo <nicolopiazzalunga@gmail.com>"
+license="custom:Proprietary"
+homepage="https://wolfram.com/mathematica"
+noshlibprovides=yes
+noverifyrdeps=yes
+nopie=yes
+nostrip=yes
+repository=nonfree
+restricted=yes
+python_version=3
+
+_M_BINARIES="MathKernel Mathematica WolframKernel math mathematica mcc wolfram"
+
+do_build() {
+	cp ${FILESDIR}/Mathematica_${version}_BNDL_LINUX.sh Mathematica.sh
+	chmod +x Mathematica.sh
+	sh Mathematica.sh -- \
+		-execdir=${DESTDIR}/usr/bin \
+		-targetdir=${DESTDIR}/opt/Mathematica \
+		-auto
+}
+
+do_install() {
+	# fix permissions
+	chmod -R go-w ${DESTDIR}/*
+	vlicense ${DESTDIR}/opt/Mathematica/LICENSE.txt
+	# remove docs
+	rm -rf ${DESTDIR}/opt/Mathematica/Documentation
```

Comment:
Why to remove docs?

Reply:
Just to save space. I made it an option.